package.json

```diff
@@ -51,5 +54,8 @@
   },
   "scripts": {
     "test": "grunt ci --verbose"
+  },
+  "browser": {
+    "pouchdb/lib/deps/request": "pouchdb/lib/deps/request-browser"
```

Comment:
But isn't that internal implementation? From my understanding, it wouldn't require any bumping, it's not something that they document anywhere, it's like using undocumented APIs on other platforms.

But maybe all this is a non-issue :) I just do it in other projects as well, and wondered if there is some common sense on this